habana/pytorch/training/docker/1.9/py3/synapseai1.1.1/Dockerfile.hpu

```diff
@@ -0,0 +1,95 @@
+FROM vault.habana.ai/gaudi-docker/1.1.1/ubuntu20.04/habanalabs/pytorch-installer:1.1.1-94
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Python versions
+ARG PYTHON=python3.8
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3.8
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    build-essential \
+    openssh-client \
+    openssh-server \
+    ca-certificates \
+    curl \
+    jq \
+    emacs \
+    git \
+    libtemplate-perl \
+    libssl1.1 \
+    openssl \
+    unzip \
+    wget \
+    vim \
+    zlib1g-dev \
+    # Install dependent library for OpenCV
+    libgtk2.0-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
+
+
+ARG PT_TRAINING_URL
+COPY ${PT_TRAINING_URL} .
+
+ARG PT_TORCHVISION_URL=https://aws-habana-pytorch-cicd-v3-binaries.s3.us-west-2.amazonaws.com/r1.9.1_aws_habana/torchvision/torchvision-0.10.1-cp38-cp38-linux_x86_64.whl
+
+RUN pip --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+# The following section uninstalls torch and torchvision before installing the
+# custom versions from an S3 bucket.
+RUN pip install --no-cache-dir --user -U \ 
+    fastai==1.0.61 \
+    "pyyaml>=5.4,<5.5" \
+    "awscli<2" \
+    psutil \
+    Pillow \
+    scipy \
+    pybind11 \
+    mpi4py==3.0.3 \
+    cmake==3.18.2.post1 \
+    torchnet \
+    "cryptography>3.2" \
+    werkzeug \
+ && pip uninstall -y dataclasses typing \
```

Comment:
@omrialmog Is this required? Can we remove this?

Reply:
@tejaschumbalkar  this can be removed. Thanks